### DISCUSSION TITLE
Increment date field

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#14805] [Q&A] [answered]
Is it possible to increment the date field? For example:

```
model Table {
  // primaryKey
  id String @id @default(uuid())

  // fields
  startDate       DateTime    @default(now()) @map("start_date")
  endDate         DateTime?   @map("end_date")
}
```

I want to increment 7 days:

```
table.updateMany({
      where: {
        startDate: {
          gte: new Date(),
        },
      },
      data: {
        startDate: {
            increment: 7
        },
      },
    });
```

**Comment by austincrim** [comment, marked as answer]
Hey @joaomoon 👋,

It is not possible to increment date fields in this way. You might be able to accomplish it using [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw), depending on your database provider.

Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you would like to see us add this in the future 👍.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
